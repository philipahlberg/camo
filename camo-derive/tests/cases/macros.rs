use camo_derive::Camo;

macro_rules! foo {
    () => { i32 }
}

#[derive(Camo)]
struct Foo {
    f: foo!(),
}

fn main() {}
