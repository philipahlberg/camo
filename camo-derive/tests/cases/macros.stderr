error: `camo` does not support macros
 --> tests/cases/macros.rs:9:8
  |
9 |     f: foo!(),
  |        ^^^
